Formula/suil.rb

```diff
@@ -40,3 +48,17 @@ def install
     system "./test"
   end
 end
+__END__
```

Comment:
```suggestion

__END__
```
We should use a patch from upstream rather than one embedded here, but if we must use an embedded patch, let's follow the style of other formulae here.

Reply:
(obsolete because patch is used)